pkg/infra/spot.go

```diff
@@ -493,7 +495,17 @@ func getMetadataMockNetworkPolicy() *networkingv1.NetworkPolicy {
 
 func getMetadataMockConfigMap() (*corev1.ConfigMap, error) {
 	// Load relative to the test execution directory
-	data, err := os.ReadFile("./infra/mock/metadata_mock.go")
+	//data, err := os.ReadFile("./infra/mock/metadata_mock.go")
```

Comment:
Leftover comment?

```suggestion
```

Reply:
Thank you @chrischdi I left it to record that I changed it. Because it is not modified for enabling OTE, but it doesn't work with OTE and has to be modified. If you think it is not needed, I can delete it.